frontend/src/containers/AdmissionsContainer/index.tsx

```diff
@@ -87,19 +95,32 @@ const AdmissionsContainer: React.FC<AdmissionsContainerProps> = ({
   );
 
   return (
-    <div>
-      <table>
+    <TableWrapper>
+      <StyledTable>
         <thead>
           {table.getHeaderGroups().map((headerGroup) => (
             <tr key={headerGroup.id}>
               {headerGroup.headers.map((header) => (
-                <th key={header.id}>
-                  {header.isPlaceholder
-                    ? null
-                    : flexRender(
+                <th key={header.id} style={{ width: header.getSize() }}>
+                  {header.isPlaceholder ? null : (
+                    <div
+                      style={
+                        header.column.getCanSort()
+                          ? { cursor: "pointer" }
+                          : undefined
+                      }
+                      onClick={header.column.getToggleSortingHandler()}
+                    >
+                      {flexRender(
                         header.column.columnDef.header,
                         header.getContext()
                       )}
+                      {{
+                        asc: " 🔼",
+                        desc: " 🔽",
```

Comment:
We really need to implement an icon library 🙈 Out of scope here though I guess

Reply:
Kinda forgot that we were already using ionicons for the rest of the project.. updated the feature to use that here as well now:)

Reply:
ah nice!